packages/mobile/src/consumerIncentives/ConsumerIncentivesHomeScreen.tsx

```diff
@@ -201,30 +206,30 @@ function ClaimSuperchargeRewards({ rewards }: { rewards: SuperchargePendingRewar
 
 export default function ConsumerIncentivesHomeScreen() {
   const { t } = useTranslation()
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    dispatch(fetchAvailableRewards())
+  }, [])
 
   const userIsVerified = useSelector((state) => state.app.numberVerified)
   const {
     hasBalanceForSupercharge,
-    token: superchargingToken,
+    superchargingToken,
     hasMaxBalance,
   } = useHasBalanceForSupercharge()
   const isSupercharging = userIsVerified && hasBalanceForSupercharge
-  const tokenToSupercharge = useTokenToSupercharge()
+  const defaultTokenToSupercharge = useDefaultTokenToSupercharge()
+  const tokenToSupercharge = superchargingToken ?? defaultTokenToSupercharge
 
-  const {
-    superchargeRewards,
-    isLoading: loadingAvailableRewards,
-    isError: errorLoadingRewards,
-  } = useFetchSuperchargeRewards()
   const claimRewardsLoading = useSelector((state) => state.supercharge.loading)
```

Comment:
small nit: I think we can do:
```
const {
 loading: claimRewardsLoading,
 availableRewards: superchargeRewards,
 fetchAvailableRewardsLoading: loadingAvailableRewards,
} = useSelector(state => state.supercharge)
```
which I think looks a bit better, but it's of course subjective